awx/main/migrations/_multi_cred.py

```diff
@@ -0,0 +1,12 @@
+def migrate_to_multi_cred(app, schema_editor):
+    Job = app.get_model('main', 'Job')
+    JobTemplate = app.get_model('main', 'JobTemplate')
+
+    for cls in (Job, JobTemplate):
+        for j in cls.objects.iterator():
+            j.credentials.add(j.credential)
```

Comment:
check `if j.credential:` to be more consistent.

This is what the corresponding workflow migration looks like:

```python
def migrate_workflow_cred(app, schema_editor):
    WorkflowJobTemplateNode = app.get_model('main', 'WorkflowJobTemplateNode')
    WorkflowJobNode = app.get_model('main', 'WorkflowJobNode')

    for cls in (WorkflowJobNode, WorkflowJobTemplateNode):
        for j in cls.objects.iterator():
            if j.credential:
                j.credentials.add(j.credential)
            j.save()
```